src/templates/README.md

```diff
@@ -0,0 +1,115 @@
+# Azure Function Templates
+
+The code in this folder is used to parse and model the [function templates](https://github.com/Azure/azure-webjobs-sdk-templates) provided by the [Azure Functions portal](https://functions.azure.com/signin). The portal provides template data in three parts: templates.json, bindingconfig.json, and resources.json. See below for an example of the schema for the TimerTrigger template
+
+## Templates.json
+[https://functions.azure.com/api/templates?runtime=~2](https://functions.azure.com/api/templates?runtime=~2)
```

Comment:
Does az create functions using v2 of the runtime?  The portal creates v1 functions.

Reply:
The templates are actually very similar in v1 vs v2. In any case, I chose v2 because that's the version of the func cli that runs on .NET Core and works on all platforms.

---
In reply to: [147805125](https://github.com/Microsoft/vscode-azurefunctions/pull/32#discussion_r147805125) [](ancestors = 147805125)